amenity-points.mss

```diff
@@ -124,8 +124,8 @@
   }
 
   [feature = 'highway_traffic_signals'][zoom >= 17] {
-    marker-file: url('symbols/traffic_light.16.svg');
-    marker-fill: #0a0a0a;
+    marker-file: url('symbols/traffic_light.svg');
+    marker-fill: mix(#0a0a0a, #ffffff, 75%);
```

Comment:
Why are we mixing instead of just setting a different colour?

Reply:
It's only a tool to make color testing much easier. Once we set the proper value, I will change it into some real value.